.lintr

```diff
@@ -0,0 +1,9 @@
+linters: with_defaults(
+    line_length_linter = NULL,
+    closed_curly_linter = NULL,
+    open_curly_linter = NULL,
+    object_name_linter = NULL,
+    object_length_linter = NULL
+  )
+exclusions: list("R/db.R" = 106)
```

Comment:
This feels fragile to me (any change to db.R before line 106) will break this.  So can we use one of the other exclusion approaches?  See `?lintr::exclude`.  It looks like this works:

```
#'
#' @keywords internal
# nolint start
verify_table <- function(con, table_name, table, identical_columns = FALSE,
                         context_info = "", solution_text = "") {
# nolint end
  if (!DBI::dbExistsTable(con, table_name)) {
    stop(sprintf(
```

Reply:
Fixed, thanks